src/navigation/PairHardwareWalletNavigator.tsx

```diff
@@ -24,16 +23,31 @@ type RouteParams = {
   PairHardwareWalletNavigatorParams: PairHardwareWalletNavigatorParams;
 };
 
-export const PairHardwareWalletNavigator = () => {
+// atoms used for navigator state
+export const LedgerImportDeviceIdAtom = atom({
+  default: '',
+  key: 'ledgerImportDeviceId',
+});
```

Comment:
Would love to see this atom moved to `src/recoil` I believe I already moved one atom from profiles files there. If there's no such directory, feel free to just create it

Reply:
i prefer this to be local, it should not be used outside of this navigator . `src/recoil` makes sense for more global atoms